### PR TITLE
Load data update

### PR DESCRIPTION
**Pull request by nathanballou**
Updates the load data examples to both remove the necessity of the "Connect Data" webpage page under "Using Saturn Cloud" and to follow the examples formatting guidelines. These will mostly serve as webpages, but conform otherwise. Will need to specifically format these to remove the "Run in Saturn Cloud" and/or code dropdown.

### CODE REVIEW

**Comment by jsignell** [comment]
~~Just wanted to say that you should feel free to put these changes off release-2021.10.26 or release-2021.11.10 if you want people to be able to see them sooner.~~

I didn't realize it was adding a new one. `main` is good.

File: examples/load-data/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Connect Saturn Cloud to Your Data
+
+Saturn does not host data, but we know that accessing your data is vital for your data science practice. We have several options for linking your data sources to your Saturn Cloud account. If your data source is not listed here, please contact us at hello@saturncloud.io or on our community Slack and we’ll help you!
```
Comment (jnolis) [inline examples/load-data/README.md]: 1. We should always say `Saturn Cloud` and never just `Saturn`
2. If people have trouble we should direct them to intercom
3. We will "sort of" host data with the shared folders, so I am gonna change the wording so we don't forget to adjust later

```suggestion
Saturn Cloud resources need to pull data from external locations and we know that accessing your data is vital for your data science practice. We have several options for loading your data sources into your Saturn Cloud resources. If your data source is not listed here and you need help connecting, please reach out using the chat option in the lower right-hand corner of any Saturn Cloud page.
```

File: examples/load-data/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## [Load Data From Snowflake](load-data-s3.ipynb)
+
+## [Load Data From Kaggle](load-data-s3.ipynb)
+
+## Next steps
+
+Thanks for trying out this resource! To learn more about how Saturn Cloud works, check out our [Documentation](https://saturncloud.io/docs/), [blog](https://saturncloud.io/s/blog/), or join an [upcoming event](https://saturncloud.io/s/events/).
+
+If you have any questions or suggestions reach out to us at support@saturncloud.io or open an issue on the [examples Github repo](https://github.com/saturncloud/examples).
```
Comment (jnolis) [inline examples/load-data/README.md]: 1. I'm not a fan of like... the advertising in the docs, ya know?
2. I don't think we should be pushing people to the repo ever.

Thus, I think we can just remove this part entirely

```suggestion
```

File: examples/load-data/load-data-local-files.ipynb [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And that's it! If you look in your resource's local file system, you will see the file.\n",
+    "\n",
+    "## Conclusion\n",
+    "\n",
+    "In this example, we showed how easy it is to copy local files to use in your Saturn Cloud notebooks.\n",
+    "\n",
+    "Want to download data from other sources? Check out our examples for downloading data from [Kaggle](./load-data-kaggle.ipynb), [S3](./load-data-s3.ipynb), and [Snowflake](./load-data-snowflake.ipynb)."
```
Comment (jnolis) [inline examples/load-data/load-data-local-files.ipynb]:
```suggestion
    "And that's it! If you look in your resource's local file system, you will see the file."
```

File: examples/load-data/load-data-kaggle.ipynb [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "That's it! Your dataset will download to your current path, and you will be able to use it for calculations!\n",
+    "\n",
+    "## Conclusion\n",
+    "In this example, we showed how easy it is to download data directly from Kaggle to use in your Saturn Cloud notebooks.\n",
+    "\n",
+    "Want to download data from other sources? Check out our examples for downloading data from [s3 buckets](./load-data-s3.ipynb) and [Snowflake](./load-data-snowflake.ipynb)."
```
Comment (jnolis) [inline examples/load-data/load-data-kaggle.ipynb]:
```suggestion
    "That's it! Your dataset will download to your current path, and you will be able to use it for calculations!"
```

File: examples/load-data/load-data-s3.ipynb [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "In this example, we showed how easy it is to download data directly from S3 to use in your Saturn Cloud notebooks.\n",
+    "\n",
+    "Want to download data from other sources? Check out our examples for downloading data from [Kaggle](./load-data-kaggle.ipynb) and [Snowflake](./load-data-snowflake.ipynb)."
+   ]
+  }
```
Comment (jnolis) [inline examples/load-data/load-data-s3.ipynb]:
```suggestion
  }
```

File: examples/load-data/load-data-snowflake.ipynb [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "In this example, we showed how easy it is to connect to Snowflake data for use in your Saturn Cloud notebooks.\n",
+    "\n",
+    "Want to download data from other sources? Check out our examples for downloading data from [Kaggle](./load-data-kaggle.ipynb) and [S3](./load-data-s3.ipynb)."
+   ]
+  }
```
Comment (jnolis) [inline examples/load-data/load-data-snowflake.ipynb]:
```suggestion
  }
```

**Review by jnolis — APPROVED** [review]
Overall I think these look good. I do have some small points of minor feedback, which I included in the notes but once fixed this can be rolled out without another round of review

(and double check my suggestions didn't break the notebook json)

**Pull request merged**